components/tooltip/index.css

```diff
@@ -59,100 +133,322 @@ governing permissions and limitations under the License.
   p {
     margin: 0;
   }
+
+  /****** Variants ******/
+  &--info,
+  &--info .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-informative, var(--spectrum-tooltip-background-color-informative));
+  }
+
+  &--positive,
+  &--positive .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-positive, var(--spectrum-tooltip-background-color-positive));
+  }
+
+  &--negative,
+  &--negative .spectrum-Tooltip-tip {
+    background-color: var(--mod-tooltip-background-color-negative, var(--spectrum-tooltip-background-color-negative));
```

Comment:
Should we add the `--highcontrast-` var here, too?

Reply:
Thanks so much for catching these mistakes!